GillianCore/debugging/debugger/debugger.ml

```diff
@@ -716,17 +716,14 @@ struct
            aux cont_func)
 
   let launch file_name proc_name =
-    let () = Fmt_tty.setup_std_outputs () in
-    let () = Config.current_exec_mode := Verification in
-    let () = PC.initialize Verification in
-    let () = Config.stats := false in
-    let () = Config.lemma_proof := true in
-    let () = Config.manual_proof := false in
-    let () =
-      match proc_name with
-      | None -> ()
-      | Some proc_name -> Config.Verification.set_procs_to_verify [ proc_name ]
-    in
+    Fmt_tty.setup_std_outputs ();
+    Config.current_exec_mode := Verification;
+    PC.initialize Verification;
+    Config.stats := false;
+    Config.lemma_proof := true;
+    (match proc_name with
```

Comment:
not a fan of `let () = ` anymore? 😛 
The only reason I use `let ()` is the `if` annoyance
```ocaml
if b then f(); g()
```
and 
```ocaml
if b then let () = f() in g()
```
mean different things and I hate it 🤦

Reply:
Hm, in that situation I'd use
```ocaml
if b then (f(); g())
```

Reply:
```ocaml
if b then begin f(); g() end
```
hahaha

Reply:
Huh, first I've heard of that, that's pretty nice. I'll use that on this situation then hah